elements/lisk-tree/src/sparse_merkle_tree/empty_node.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { EMPTY_HASH } from './constants';
+
+export class Empty {
+	private readonly _hash: Buffer;
```

Comment:
You can directly assign`EMPTY_HASH` to the attribute directly.  if you prefer so. Then no need of constructor. 
```suggestion
	private readonly _hash: Buffer = EMPTY_HASH;
```